spec/models/adviser_spec.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.describe Adviser, type: :model do
```

Comment:
Perhaps this test should be updated to test `lib/ext/adviser.rb` and more explicitly test the outcome of that method.